docs/content/patterns/alz/Cleaning-Up-Preview-Version-Deployment.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cleaning up preview version deployment
+geekdocCollapseSection: true
+weight: 70
+---
+
+In some scenarios, it may be necessary to remove everything deployed by the ALZ Monitor solution. The instructions below detail execution of a PowerShell script to delete all resources deployed, including:
+
+- Metric Alerts
+- Activity Log Alerts
+- Resource Groups (created for to contain alert resources)
+- Policy Assignments
+- Policy Definitions
+- Policy Set Definitions
+- Policy Assignment remediation identity role assignments
+
+All resources deployed as part of the initial ALZ Monitor deployment and the resources created by dynamically by 'deploy if not exist' policies are either tagged, marked in metadata, or in description (depending on what the resource supports) with the value `_deployed_by_alz_monitor` or `_deployed_by_alz_monitor=True`. This metadata is used to execute the cleanup of deployed resources; _if it has been removed or modified the cleanup script will not include those resources_.
+
+## Cleanup Script Execution
```

Comment:
Please add a comment that the script should be thoroughly tested before uisng in production environments and we assume no responsibility in the event of resource or data loss (I think Jan has something he has written previously) and there is one for ALZ clean-up that has something in the comments.